pymitsubishi/mitsubishi_controller.py

```diff
@@ -38,16 +37,11 @@ def create(cls, device_ip: str, port: int = 80, encryption_key: str | bytes = "u
         api = MitsubishiAPI(device_ip=device_ip, port=port, encryption_key=encryption_key)
         return cls(api)
 
-    def fetch_status(self, detect_capabilities: bool = True) -> bool:
+    def fetch_status(self) -> bool:
```

Comment:
You need to update documentation in README.md if you remove this part

Reply:
Good catch, thx!